LearningMissionSimulation/PlaygroundEdvard.cs

```diff
@@ -47,6 +54,77 @@ public static void Action1()
             //unit0.Report();
         }
 
+        public void Action2(int accountCount)
+        {
+            CreateAccounts(accountCount);
+            ActivateAccounts();
+        }
+
+        public void CreateAccounts(int accountCount)
+        {
+            for (int i = 0; i < accountCount; i++)
+            {
+                Console.WriteLine($" Account {i + 1}");
+
+                createdAccountList.Push(ObjectGenerator.GenerateAccount());
+
+                Console.WriteLine('\n');
+            }
+
+            foreach (Account account in createdAccountList)
+            {
+                if (account.Role == Role.Student)
+                {
+                    studentList.Add(ObjectGenerator.GenerateStudent(account.Id));
+                }
+                else if (account.Role == Role.Instructor)
+                {
+                    instructorList.Add(ObjectGenerator.GenerateInstructor(account.Id));
+                }
+            }
+        }
+
+        // Unfair Coordinator
+        public void ActivateAccounts()
+        {
+            Account lastAccount = createdAccountList.Peek();
+
+            if (lastAccount.Role == Role.Student && lastAccount.Status == Status.Pending)
```

Comment:
Your method only activates the last account and only if it is student or instructor. This is not what I expected to see. Your method should activate all students and instructor accounts but in the Last Come First Served order.